linera-ethereum/Cargo.toml

```diff
@@ -17,16 +17,23 @@ features = ["ethereum"]
 ethereum = []
 
 [dependencies]
-alloy.workspace = true
 alloy-primitives.workspace = true
 anyhow.workspace = true
-linera-storage-service.workspace = true
+async-trait.workspace = true
 num-bigint.workspace = true
 num-traits.workspace = true
 serde.workspace = true
+serde_json = { workspace = true, features = ["raw_value"] }
 thiserror.workspace = true
-tokio = { workspace = true, features = ["full"] }
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+alloy = { workspace = true, default-features = false, features = [ "providers", "rpc-types-eth", "reqwest", "rpc-client", "provider-http", "json-rpc", "signers", "sol-types", "node-bindings", "transports", "transport-http", "signer-wallet", "network", "json-abi", "json-rpc", "contract" ] }
```

Comment:
That's a lot of features. :sweat_smile: 
Are we using all of those? And there are still other default features that we're not using?

Reply:
I prefer not to use the default features.
I reduce the number of features used, but some are implied by other features.